CppSamples/Analysis/AnalyzeHotspots/AnalyzeHotspots.cpp

```diff
@@ -18,22 +18,25 @@
 #include "pch.hpp"
 #endif // PCH_BUILD
 
+// sample headers
 #include "AnalyzeHotspots.h"
 
-#include "Map.h"
-#include "MapQuickView.h"
-#include "MapTypes.h"
-#include "GeoprocessingTask.h"
+// C++ API headers
```

Comment:
This is picky, but "C++ API" is how we internally differentiate between our internal API and our public-facing API. It's a fairly generic phrase for samples, so I would recommend using "ArcGIS Maps SDK headers" or something like that in the samples repo only.